COPYRIGHT.txt

```diff
@@ -1349,7 +1349,7 @@ License: FTL
      Discusses bugs,  as well  as engine internals,  design issues,
      specific licenses, porting, etc.
  .
-   o http://www.freetype.org
+   o https://www.freetype.org
```

Comment:
Are we using an old version of the FTL?
https://www.freetype.org/license.html
The last update seems `2006-Jan-27`, while we have `2000-Feb-08`. Should be double checked